howtos/work_with_pc_files.rst

```diff
@@ -0,0 +1,169 @@
+.. _pc_files:
+
+Work with pkg-config and pc files
+=================================
+
+If you are creating a Conan package for a library (A) and the build system uses ``.pc`` files to locate
+its dependencies (B and C), Conan packages too, you can follow different approaches.
+
+The main issue to solve is the absolute paths. When an user installs a package in the local cache,
+the directory will probably be different from the directory where the package was created,
+because of the different computer, conan home directory or even different user or channel:
+
+In the machine where the packages were created:
+
+.. code-block:: text
+
+    /home/user/lasote/.data/storage/zlib/1.2.11/conan/stable
+
+In the machine where some user are reusing the library:
+
+.. code-block:: text
+
+    /custom/dir/.data/storage/zlib/1.2.11/conan/testing
+
+So the ``.pc``  files containing absolute paths won't work to locate the dependencies.
+
+
+Example of a ``.pc`` file with an absolute path:
+
+
+.. code-block:: text
+
+    prefix=/Users/lasote/.conan/data/zlib/1.2.11/lasote/stable/package/b5d68b3533204ad67e01fa587ad28fb8ce010527
+    exec_prefix=${prefix}
+    libdir=${exec_prefix}/lib
+    sharedlibdir=${libdir}
+    includedir=${prefix}/include
+
+    Name: zlib
+    Description: zlib compression library
+    Version: 1.2.11
+
+    Requires:
+    Libs: -L${libdir} -L${sharedlibdir} -lz
+    Cflags: -I${includedir}
+
+
+Approach 1: Import and patch the prefix in the `pc` files
+---------------------------------------------------------
+
+Following this approach your library `A` will import to a local directory the ``.pc`` files from `B` and `C`, then,
+as they will contain absolute paths, the recipe for `A` will patch the paths to match the current installation
+directory.
+
+You will need to package the `pc` files from your dependencies.
+You can adjust the `PKG_CONFIG_PATH` to let ``pkg-config`` tool locate your ``.pc`` files.
+
+.. code-block:: python
+
+    import os
+    from conans import ConanFile, tools
+
+    class LibAConan(ConanFile):
+        name = "libA"
+        version = "1.0"
+        settings = "os", "compiler", "build_type", "arch"
+        exports_sources = "*.cpp"
+        requires = "libB/1.0@conan/stable"
+
+        def build(self):
+            from conans.client.file_copier import FileCopier
+            lib_b_path = self.deps_cpp_info["libB"].rootpath
+            file_copier = FileCopier(lib_b_path, ".")
```

Comment:
IMHO, using ``FileCopier`` is a bit obscure here. Why not trying a bare ``shutil.copy()``?